### PR TITLE
Add tests for extract-device-id issue, update kitchen test suite

### PR DESCRIPTION
**Pull request by drenalin23**
Update include_recipe ordering in default.rb to get maas_service_uri attribute to load correctly.

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -62,9 +61,9 @@
   include_recipe "repose::service-#{service}"
 end
 
-if %w(stage prod).include?(node.chef_environment)
+if node.chef_environment.include?('stage')
```
Comment (jjbuchan) [inline recipes/default.rb]: Why's this only happen for stage?

Comment (mburns) [inline recipes/default.rb]: this change intended? Not sure I follow

File: recipes/default.rb [hunk 2]
```diff
@@ -62,9 +61,9 @@
...
   # set non-default (environment-specific) configuration
-  node.default['repose']['extract_device_id']['maas_service_uri'] = "http://#{node['networks']['ipaddress_eth0']}:7000"
+  node.default['repose']['extract_device_id']['maas_service_uri'] = "http://#{node['networks']['ipaddress_eth0']}:7000"
```
Comment (mburns) [inline recipes/default.rb]: trailing whitespace :)

**Comment by drenalin23** [comment]
@jjbuchan @mburns shouldn't you guys be like sleeping or something?  commits should be all good now - give a second read through.  version bump included in commit 2.

**Pull request merged**